assets/js/just-the-docs.js

```diff
@@ -463,23 +466,66 @@ jtd.setTheme = function(theme) {
 
 // Scroll site-nav to ensure the link to the current page is visible
 
+// Note: pathname can have a trailing slash on a local jekyll server
+// and not have the slash on GitHub Pages
+
 function scrollNav() {
-  const href = document.location.pathname;
+  var href = document.location.pathname;
+  if (href.endsWith('/') && href != '/') {
+    href = href.slice(0, -1);
+  }
```

Comment:
nit: seeing that this is repeated a couple times, perhaps it could be extracted into a helper?

Reply:
I was trying to minimise changes to `scrollNav`. And I'd prefer to leave such factoring-out to those with more JS experience than I have…

Reply:
All good, this can be a later project then!